origin-growth/src/resources/email.js

```diff
@@ -47,13 +50,14 @@ async function sendInvites(referrer, recipients) {
     }
 
     // Send the invite code to the recipient.
-    // TODO: should we have an html version of the email ?
+    const dappUrl = process.env.DAPP_URL || 'http://localhost:3000'
+    const welcomeUrl = `${dappUrl}?inviteCode=${code}`
```

Comment:
Currently in url pattern for invite code is: 
`http://{dapp-url}/#/welcome/[invite-code]` e.g.
`http://localhost:3000/#/welcome/msjcfgevu2`

Reply:
Done